source/options.tsx

```diff
@@ -173,10 +174,37 @@ async function highlightNewFeatures(): Promise<void> {
 	void browser.storage.local.set({featuresAlreadySeen});
 }
 
+async function getFeaturesDisabledViaHotfix(): Promise<HTMLElement[]> {
+	const hotfixes = await cache.get<string[][]>('hotfixes');
+	if (!hotfixes) {
+		return [];
+	}
+
+	const {version} = browser.runtime.getManifest();
+	const featureNames: Set<string> = new Set(features.map(({id}) => id));
+	const disabledFeatures = [];
+	for (const [feature, unaffectedVersion] of hotfixes) {
+		const relatedIssue = '#1234';
+		if (featureNames.has(feature) && (version === '0.0.0' || !unaffectedVersion || compareVersions(unaffectedVersion, version) > 0)) {
+			disabledFeatures.push(
+				<p>
+					<code>{feature}</code> has been temporarily disabled
+					{relatedIssue ? <> due to <a href={'https://github.com/sindresorhus/refined-github/issues/' + relatedIssue.slice(1)}>{relatedIssue}</a></> : false}.
```

Comment:
We control the data, let's never use `.slice(1)` to drop `#` after _we_ add it. 
```suggestion
					{relatedIssue ? <> due to <a href={'https://github.com/sindresorhus/refined-github/issues/' + relatedIssue}>#{relatedIssue}</a></> : false}.
```

Also perhaps we need a global utility function to generate issue links to our repo

Reply:
> Also perhaps we need a global utility function to generate issue links to our repo

Same PR?